scripts/integration-test.sh

```diff
@@ -1,6 +1,531 @@
-#! /bin/bash -e
+#!/bin/bash
```

Comment:
-e means abandon the script if any command returns errors with a value other than 0.  I want to detect and deal with errors so I can do cleanup rather than just abort.